app/controllers/shopping_lists_controller.rb

```diff
@@ -0,0 +1,16 @@
+class ShoppingListsController < ApplicationController
+  def index
+    @recipes = current_user.recipes
+    @total_missing_count = 0
+    @total_missing_price = 0
+    @missing_foods = []
+    @recipe_foods = RecipeFood.where(recipes: @recipes)
+
+    @recipe_foods.each do |recipe_food|
+      missing_quantity = recipe_food.food.quantity - recipe_food.quantity
+      @missing_foods << recipe_food
+      @total_missing_count += 1
+      @total_missing_price += missing_quantity * recipe_food.food.price
+    end
+  end
+end
```

Comment:
- [ ] Great job fixing the error in the shopping list as requested, however now that the shopping list is visible, we need to ensure that the total of the list is displayed at the top right corner, and the quantity and price of each item on the list should be visible.

Your Shopping List
![image](https://github.com/chiztechnology/rails-capstone-project/assets/81932478/a95758cb-eaac-4cc0-91eb-e8e03f2a607e)

Given Wireframe
![image](https://github.com/chiztechnology/rails-capstone-project/assets/81932478/d2bfcafc-1516-4ee9-8abc-77a828bdb300)
##### Please Note that we have to implement all functionality included in the requirements, and because the app wasnt running there was no way to request this change up until now

Reply:
I understand it. is it possible to get some help regarding this issue? I'm still stuck on it.

Reply:
hello dear @Munsa1 , can you provide some help on how to fix this please ?